theme/layout/theme.liquid

```diff
@@ -20,5 +20,22 @@
   </main>
     {% render 'footer' %}
     {{ 'index.js' | asset_url | script_tag }}
+
+    <script type="text/javascript" src="//code.jquery.com/jquery-1.11.0.min.js"></script>
+    <script type="text/javascript" src="//code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
+    {{ 'slick.min.js' | asset_url | script_tag }}
+
+    <script type="text/javascript">
+      jQuery( document ).ready(function() {
+        $('.ProductImagesSlider__slider').slick({
+            dots: false,
+            autoplay: false,
+            arrows: false,
+            infinite: false,
+            slidesToShow: 1.2,
+            centerMode: true,
+        });
+      });
```

Comment:
I think XXIX will probably want the first image to be more centered, but let's see what they think in QA. :) Right now it's being pushed left by the next slide so it's not 100% centered, since I'm showing 1.2 slides instead of 1 slide.